### PR TITLE
K8s: Remove /k8s/ prefix

### PR DESCRIPTION
**Pull request by toddtreece**
**What is this feature?**

Currently the k8s apiserver is mounted at `localhost:3000/k8s/apis`. This change removes the `/k8s/` prefix from the path. The new URL will be: `localhost:3000/apis`

**Why do we need this feature?**

Removing the `/k8s/` prefix will mount the kubernetes compatible APIs at `/apis/`, which is the same path that a standard standalone apiserver would use. 

**Who is this feature for?**

@grafana/grafana-app-platform-squad 

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by ryantxu** [comment]
This is great!  the other path we need to expose is `openapi`

**Review by DanCech — APPROVED** [review]

File: pkg/services/grafana-apiserver/service.go [hunk 1]
```diff
@@ -141,7 +140,13 @@ func ProvideService(
...
 		k8sRoute.Any("/", middleware.ReqSignedIn, handler)
 		k8sRoute.Any("/*", middleware.ReqSignedIn, handler)
-	})
+	}
+
+	s.rr.Group("/apis", proxyHandler)
+	s.rr.Group("/openapi", proxyHandler)
+	s.rr.Group("/healthz", proxyHandler)
+	s.rr.Group("/livez", proxyHandler)
+	s.rr.Group("/readyz", proxyHandler)
```
Comment (toddtreece) [inline pkg/services/grafana-apiserver/service.go:149]: @ryantxu i added `openapi` and the other top level k8s routes

Comment (ryantxu) [inline pkg/services/grafana-apiserver/service.go:149]:
```suggestion
```
I think we can punt on exposing these

**Review by ryantxu — APPROVED** [review]

**Review by mildwonkey — APPROVED** [review]
:shipit:

**Pull request merged**